### PR TITLE
Updated app and action metadata.

### PR DESCRIPTION
**Pull request by govind-phantom**
Hey Ryan,

I have made some modifications to your JSON file so that it meets our requirements for releasing in our app store and the modified version of the app should be available in a few days. This should not require any code changes. Thanks for writing this app for us!
## 

Changed app type to one of the existing categories.
Removed contains types for generic data.
Change send to google drive action to "upload file".
Change trigger ifttt action to "run action".

Signed-off-by: Govind Salinas govind@phantom.us

### CODE REVIEW

File: ifttt/ifttt.json [hunk 1]
```diff
@@ -78,21 +78,21 @@
...
+            "contains": [],
             "required": false
           }
         },
         "output" : [],
         "versions":"EQ(*)"
       },
       {
-        "action": "save to google drive",
+        "action": "upload file",
```
Comment (kranzrm) [inline ifttt/ifttt.json:95]: This really is not uploading a file, it is appending information to a spreadsheet.  Would there be a better name for this?

File: ifttt/ifttt.json [hunk 2]
```diff
@@ -49,19 +49,19 @@
           "value1": {
             "description": "First Optional Value",
             "data_type": "string",
-            "contains": ["domain", "malwr task id", "ip", "pid", "vm", "srp guid", "file size", "vault id", "firewall rule name", "carbon black query type", "port", "carbon black sensor id", "process name", "cyphort event id", "tanium question", "isightpartners report id", "flash", "pe file", "network application", "threatgrid task id", "jira project key", "rt ticket id", "sha256", "email", "host name", "carbon black query", "hash", "anubis task id", "rt queue", "javascript", "urlquery report id", "volatility profile", "os memory dump", "file path", "qradar offense id", "urlquery queue id", "carbon black watchlist", "servicenow ticket id", "md5", "mobileiron device uuid", "wepawet task id", "sha1", "lastline task id", "mac address", "url", "doc", "file name", "user name", "cuckoo task id", "jira ticket status", "pdf", "jira ticket key"],
+            "contains": [],
```
Comment (kranzrm) [inline ifttt/ifttt.json:52]: I was using this field to allow the user to select any known value to automatically be put into this field.  For example, in the GUI playbook editor and the dropdown menu in the incidents.  

Is there a preferred way to allow this?  
Is the GUI going to be a part of your long-term vision?

**Comment by kranzrm** [comment]
Hi Govind,

Thanks for the changes.  I left a couple questions inline.  I don't have any problem with any of them, just wanted some clarification.  Let me know if you'd like to setup a quick conversation on slack.

Thanks,
Ryan Kranz

**Comment by govind-phantom** [comment]
Hey Ryan,

Thanks for getting back to me. Let me try to answer both your questions
here.
1. We had toyed with "cloud upload" and "send to" since we try to keep the
   action to 2 words and we would like to have the action names be more
   generic so that we reuse the name between similar services. So "upload to
   google drive" was too long and too specific. Do you prefer either of the
   options I stated, or if you have another idea I would like to hear it.
2. The problem with a list of contains like this is that it basically shows
   up on everywhere. It is so ubiquitous that we think it will cause some
   confusion for the user. If you feel strongly about it we can discuss it
   further. And, yes, we want a robust GUI that offers interoperability
   between apps the main problem is that we had not considered an action that
   takes anything as its input and it is a bit of an awkward fit.

I hope that answers your questions. Let me know what the final changed you
would like to make are so we get a final version out.

-Govind

**Comment by kranzrm** [comment]
Govind,

Thanks again for your input.  I see your predicament now.  I'll go with _send to_ and stand by until you have a better way to address actions that could be conducted across all fields

Thanks,
Ryan

**Pull request merged**